fsspec/implementations/tests/test_local.py

```diff
@@ -439,3 +440,11 @@ def test_auto_mkdir_warns():
     LocalFileSystem.clear_instance_cache()
     with pytest.warns(FutureWarning, match="auto_mkdir=True has been deprecated"):
         LocalFileSystem()
+
+
+def test_strip_protocol_expanduser():
+    path = "file:///C:\\foo\\bar" if sys.platform == "win32" else "file://~/foo/bar"
```

Comment:
I don't have access to a windows machine, so I haven't been able to test that `"file:///C:\\foo\\bar"` passes locally

Reply:
Shouldn't "~" work on windows too?
Also, I notice there are three "/" characters  in the win path, not two.

Reply:
Oops my mistake, I forgot to include the `"~"` there. I'll add it now.

I added the extra "/" based on 

https://github.com/intake/filesystem_spec/blob/c325a40e5c3bb73100c0f922bccf2e0d4e8bca3a/fsspec/implementations/tests/test_local.py#L84

Should the protocol always be `file://` (two slashes)?

Reply:
I don't know why there was three there. I hope two is right... Let's get windows tests on this. I can do it sometime not today.